ebos/eclnewtonmethod.hh

```diff
@@ -159,16 +159,25 @@ public:
             }
 
             const auto& r = currentResidual[dofIdx];
-            const double pvValue =
+            Scalar pvValue =
```

Comment:
why not const here?

Reply:
because the variable is potentially useful for modification in the function and honestly `const` is IMO slightly ugly while not being useful for by-value objects.

Reply:
> because the variable is potentially useful for modification in the function and honestly

when you need to change it, you remove `const`, instead of the other way around.

Reply:
... or just never use `const` for by-value objects. this is the approach 90% of C++ code seems to use ;)

Reply:
> this is the approach 90% of C++ code seems to use ;)

I am really not sure whether this statistics exist, while you do not have a good reason to remove the existing `const`s here.